Patch to downrev the required version of pbr to something we have.
This patch is not suitable for upstream commitment.

--- cloudbase-init-0.9.7/setup.py.orig	2014-12-22 09:21:29.644355381 -0700
+++ cloudbase-init-0.9.7/setup.py	2014-12-22 09:21:38.592302541 -0700
@@ -17,5 +17,5 @@
 import setuptools
 
 setuptools.setup(
-    setup_requires=['pbr>=1.3'],
+    setup_requires=['pbr>=0.8.1'],
     pbr=True)
